File: palindrom.java
import java.util.Scanner;

public class palindrom {
      public static void main(String[] args) {
            int no,real,rev=0;
            System.out.println("\nEnter no to check Palindrome : ");
            Scanner sc = new Scanner(System.in);
            no = sc.nextInt();
            real = no;
            while (no>0)
            {
                  int rem = no%10;
                  no/=10;
                  rev=rev*10+rem;
            }
            if(rev==real) System.out.println("Yes");
            else    System.out.println("NO");
      }
}

File: palindrom.cpp
#include <iostream>
using namespace std;

int main()
{
   int no,real,rev=0;
   cout<<"\nEnter no to check Palindrome : ";
   cin>>no;
   real = no;
   while (no>0)
   {
      int rem = no%10;
      no/=10;
      rev=rev*10+rem;
   }
   if(rev==real) cout<<"Yes";
   else    cout<<"NO";

return 0;
}